### PR TITLE
[DOCS] DynamicFont

### PR DESCRIPTION
**Pull request by diowonderland**

### CODE REVIEW

File: doc/classes/DynamicFont.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="DynamicFont" inherits="Font" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		DynamicFont renders [code].ttf[/code] and [code].otf[/code] font files.
 	</brief_description>
 	<description>
+		DynamicFont renders fonts from [code].ttf[/code] and [code].otf[/code] font files dynamically during runtime using FreeType instead of using a prerendered texture atlas like [BitmapFont]. This trades the faster loading time of [BitmapFont]\ s for the ability to change font parameters like size and spacing during runtime. [DynamicFontData] is used for referencing the font file paths.
```
Comment (djrm) [inline doc/classes/DynamicFont.xml]: good description however i would suggest to say `renders vector font files (such as TTF or OTF)  dynamically`, also i would not include `using freetype`

Comment (cbscribe) [reply doc/classes/DynamicFont.xml]: I agree with @djrm

**Review by cbscribe — CHANGES_REQUESTED** [review]
Good work - thanks!

**Comment by diowonderland** [comment]
Updated!

**Pull request merged**